stellarphot/core.py

```diff
@@ -166,9 +166,15 @@ def _update_colnames(self, colname_map, data):
     def _update_passbands(self):
         # Converts filter names in filter column to AAVSO standard names
         # Assumes _passband_map is in namespace.
-        for orig_pb, aavso_pb in self._passband_map.items():
-            mask = self["passband"] == orig_pb
-            self["passband"][mask] = aavso_pb
+
+        # Create a list of new passband names instead of trying to change names in place
```

Comment:
Took me a bit of effort to comprehend the list comprehension... but I think it is sound.